command/start.go

```diff
@@ -43,7 +44,7 @@ type startCommand struct {
 	autocompleteInstall   *bool
 	autocompleteUninstall *bool
 
-	isDefault bool
+	isDefault *bool
```

Comment:
I was initially wondering why a flag was used. If I understand correctly, a flag is used because it is the easiest (only?) way to pass the default information to the command.

Let me know if my understanding is incorrect. Either way, would it be possible to add a quick comment with this context?

Reply:
yep, you're understanding correctly, this is the easiest (probably only?) way I could pass this information

Reply:
Gotcha, thanks for confirming